distribution/docs/src/main/resources/content/_configuring/query-feedback.adoc

```diff
@@ -0,0 +1,70 @@
+:title: Configuring Query Feedback for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 04
+:summary: Configuring Query Feedback for ${catalog-ui}.
+
+== {title}
+
+${catalog-ui} provides an option to allow users to submit Query Feedback.
+
+.Configuring Query Feedback
+. First, configure the *Email Service* to point to a mail server. See <<org.codice.ddf.platform.email.impl.SmtpClientImpl, Email Service Configurations>>.
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-ui}* application.
+. Select the *Configuration* tab.
+. Select *Catalog UI Search*.
+. Select the *Enable Query Feedback* option to enable the query comments option for users in ${catalog-ui}.
+. Add a *Query Feedback Email Subject Template*.
+. Add a *Query Feedback Email Body Template*. The template may include HTML formatting.
+. Add the *Query Feedback Email Destination*.
+. Select the *Save* button.
+
+.Query Feedback Template Replacements
+
+The following keywords in the templates will be replaced with submission-specific values, or "Unknown" if unknown.
+
+[cols="1m,1"]
+|===
+|Template keyword |Replacement value
+
+|`{{auth_username}}`
+|Username of the security subsystem
```

Comment:
this is more a future thing, but I wonder if we should have a section about this security subsystem that we could link to?